testData/preferences/maria.json5

```diff
@@ -0,0 +1,47 @@
+// # Maria.json5
+//
+// This preference set changes the Windows 10 magnifier preferences settings.
+//
+// ## Testing
+//
+// This changes the following settings from the system:
+//     + FollowFocus: Narrator doesn't follow focus.
+//     + FollowCaret: Narrator now follows text insertion point.
+//     + Invert: Narrator invert the colours.
+//     + Magnification: Magnification is set to 150%.
+//     + MagnificationMode: Magnification mode is set to "Lens mode".
```

Comment:
Isn't it actually the case that it is set to "Docked"?

Reply:
Nice catch! Changing it.